src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -29,6 +29,10 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
 
   printedName = "easy-stage-file-item"
   version(s"$printedName v${Version()}")
+
+  editBuilder(_.setHelpWidth(100))
+  val NB_SPACE = "\u00A0" // &nbsp; won't work when calling for help on a console
```

Comment:
That was what I meant but original defaults have been restored in scallop-V1.0.1 so I expect we can get rid of this again.